entries/p/poeditor.com.json

```diff
@@ -0,0 +1,11 @@
+{
+  "POEditor": {
+    "domain": "poeditor.com",
+    "tfa": [
+      "totp"
+    ],
```

Comment:
```suggestion
    ],
    "documentation": "https://poeditor.com/kb/how-to-enable-two-factor-authentication",
```

Reply:
I don't want to know why I didn't find that entry -.-, Thanks Michael!